### DISCUSSION TITLE
Does this support docker swarm?

### DISCUSSION THREAD

**Discussion by trajano** [abiosoft/colima#469] [Q&A] [answered]
Does this support docker swarm?  It may be useful to add to the FAQ.

**Comment by abiosoft** [comment, marked as answer]
Yeah, though assigning a network address is required for the services (for now).

```
colima start --network-address
```

use the IP address from `colima ls` as advertise address

```
 docker swarm init --advertise-addr 192.168.106.2
```
access the services via the IP address

```
curl http://192.168.106.2:8080
```

**Reply by genki** [reply]
@abiosoft 
It seems that currently the `colima ls` doesn't show IP address as follows:
```
$ colima ls -v
PROFILE    STATUS     ARCH      CPUS    MEMORY    DISK      RUNTIME    ADDRESS
default    Running    x86_64    4       4GiB      100GiB    docker     
worker     Running    x86_64    2       2GiB      20GiB     docker     
```
I am using `colima version 0.5.4`
